js/main.js

```diff
@@ -161,6 +167,12 @@ function directorInit() {
   $('.js-load-overlay-vimeo').on('click', function () {
     overlayVimeoPlayer.load($(this).data(), $(this).index());
   });
+    
+    // LOAD STILLS IN OVERLAY
+
+  $('.js-load-overlay-image').on('click', function () {
+    overlayImage.load($(this).children('img').clone(), $(this).index());
```

Comment:
hmm but the src-set is based off the size specified by that image. which is 'still' or something and not a particularly large crop right?